src/v2/Apps/Artist/Routes/WorksForSale/Components/ArtistArtworkFilter.tsx

```diff
@@ -34,6 +37,14 @@ const ArtistArtworkFilter: React.FC<ArtistArtworkFilterProps> = props => {
   const trendingSortVariant = useFeatureVariant(
     "trending-sort-for-artist-artwork-grids"
   )
+  const { trackFeatureVariant } = useTrackFeatureVariant({
+    experimentName: "trending-sort-for-artist-artwork-grids",
+    variantName: trendingSortVariant?.name!,
+  })
+
+  React.useEffect(() => {
```

Comment:
**thought(super minor):** noticed that [here](https://github.com/artsy/force/blob/b5327a4ca011d0940e8dabbd43d4b3e3441be5f0/src/v2/Apps/Artist/Routes/WorksForSale/Components/ArtistArtworkFilter.tsx#L12) we are importing `* as React from 'react'`, we could just import the `useEffect` and use it like this and remove the import on [this line](https://github.com/artsy/force/blob/b5327a4ca011d0940e8dabbd43d4b3e3441be5f0/src/v2/Apps/Artist/Routes/WorksForSale/Components/ArtistArtworkFilter.tsx#L12) completely. Up to you though it is super minor 🎉
```suggestion
  useEffect(() => {
```